### PR TITLE
issue added dynamic form library

### PR DESCRIPTION
**Pull request by GSuma21**
Ui for card , filter and search according to products updates and added missing ui for resource holder component

### CODE REVIEW

File: projects/lib-shared-modules/src/lib/components/card/card.component.scss [hunk 1]
```diff
@@ -18,15 +20,23 @@
...
     padding: 0;
-    
+}
+.overflow-ellipsis{
+    overflow: hidden;
+    text-overflow:ellipsis;
+    white-space: nowrap;
+    white-space: pre-line;
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
```
Comment (ramkumar-ns-dev) [inline projects/lib-shared-modules/src/lib/components/card/card.component.scss:38]: why we need this ?

File: projects/self-creation-portal/src/app/components/create-new/create-new.component.ts [hunk 2]
```diff
@@ -71,36 +71,36 @@ export class CreateNewComponent {
...
-  openPopup() {
-    const dialogRef = this.dialog.open(DialogPopupComponent, {
-    data : {
-      header: 'BACK',
-      content: 'CHANGE_UNSAVED_MESSAGE',
-      cancelButton:'CANCEL',
-      exitButton: 'EXIT'
-    }
-    });
+  // openPopup() {
```
Comment (ramkumar-ns-dev) [inline projects/self-creation-portal/src/app/components/create-new/create-new.component.ts]: why it's commented?

**Review by ramkumar-pacewisdom — CHANGES_REQUESTED** [review]
please change styles from pixels to "rem or em" or percentage.

File: projects/lib-shared-modules/src/lib/components/card/card.component.scss (hunk 1 above)
Comment (GSuma21) [reply projects/lib-shared-modules/src/lib/components/card/card.component.scss:38]: it will handle 1 line and width according to card size

**Review by ramkumar-pacewisdom — APPROVED** [review]

**Review by ramkumar-pacewisdom — APPROVED** [review]

**Pull request merged**